README.md

```diff
@@ -87,16 +86,18 @@ This project supports below scenarios for end-to-end guest operating system vali
 | UnionTech OS Server 20 1050a                    | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | Fedora Server 36 and later                      | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
 | ProLinux Server 7.9, 8.5                        | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
-| Pardus 21.2 Server and later                    | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
-| Pardus 21.2 XFCE Desktop and later              | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
+| FreeBSD 13 and later (*)                        | :heavy_check_mark:               |                          |                                    |
+| Pardus 21.2 Server,XFCE Desktop and later (*)    | :heavy_check_mark:               |                          | :heavy_check_mark:                 |
```

Comment:
The last ":heavy_check_mark: " should be removed since Pardus does not supprt test cases.

Reply:
Yes, you are right. Removed it. Thanks!